linkml-schema/reproschema.yaml

```diff
@@ -0,0 +1,728 @@
+  additionalNotesObj:
+    title: additional notes
+    comments:
+    - A set of objects to define notes in a field. For example, most Redcap and NDA
+      data dictionaries have notes for each item which needs to be captured in reproschema.
+    slot_uri: reproschema:additionalNotesObj
+    multivalued: true
+    any_of:
+    - range: AdditionalNoteObj
+  allow:
+    title: allow
+    comments:
+    - 'An array of items indicating properties allowed on an activity or protocol '
+    slot_uri: reproschema:allow
+    multivalued: true
+    any_of:
+    - range: Thing
+  altLabel:
+    title: alternate label
+    comments:
+    - The alternate label.
+    slot_uri: skos:altLabel
+  associatedMedia:
+    title: associatedMedia
+    comments:
+    - A media object that encodes this CreativeWork. This property is a synonym for
+      encoding.
+    slot_uri: schema:associatedMedia
+  audio:
+    title: audio
+    comments:
+      - TDOO
+    slot_uri: schema:audio
+    range: uriorcurie
+    inlined: true
+  choices:
+    title: choices
+    comments:
+    - An array to list the available options for response of the Field item.
+    exact_mappings:
+    - schema:itemListElement
+    slot_uri: reproschema:choices
+    multivalued: true
+    any_of:
+    - range: uri
+    - range: Choice
+  citation:
+    slot_uri: schema:citation
+  column:
+    title: column
+    comments:
+    - An element to define the column name where the note was taken from.
+    slot_uri: reproschema:column
+    multivalued: true
+    any_of:
+      - range: string
+      - range: langString
+  compute:
+    title: computation
+    comments:
+    - 'An array of objects indicating computations in an activity or protocol and
+      maps it to the corresponding Field item. scoring logic is a subset of all computations
+      that could be performed and not all computations will be scoring. For example,
+      one may want to do conversion from one unit to another. '
+    slot_uri: reproschema:compute
+    multivalued: true
+    any_of:
+    - range: ComputeSpecification
+  contentUrl:
+    slot_uri: schema:contentUrl
+    range: uriorcurie
+  cronTable:
+    title: cronTable
+    comments:
+      - TODO not described in reproschema
+    slot_uri: reproschema:cronTable
+  datumType:
+    title: datumType
+    comments:
+    - Indicates what type of datum the response is (e.g. range,count,scalar etc.)
+      for the Field item.
+    slot_uri: reproschema:datumType
+    any_of:
+    - range: string
+    - range: uri
+  description:
+    slot_uri: schema:description
+  endedAtTime:
+    range: datetime
+    slot_uri: prov:endedAtTime
+  generated:
+    slot_uri: prov:generated
+  image:
+    title: image
+    comments:
+    - An image of the item. This can be a <a class="localLink" href="http://schema.org/URL">URL</a>
+      or a fully described <a class="localLink" href="http://schema.org/ImageObject">ImageObject</a>.
+    slot_uri: schema:image
+    range: uriorcurie
```

Comment:
so you would like to add `imageURL` as a separate slot?

For the future, I understand you would like to have something like this?
```
image:
  any_of:
  - range: uriocurie
  - range: ImageObject
```

Reply:
right now there is som inconsistency (perhaps partially introduced by myself), in a way how different media are in the model, see for comparison [audio](https://github.com/ReproNim/reproschema/blob/b5474b89aceb841cdfdbb0ea51d30e1389f301a2/linkml-schema/reproschema.yaml#L60) and [video](https://github.com/ReproNim/reproschema/blob/b5474b89aceb841cdfdbb0ea51d30e1389f301a2/linkml-schema/reproschema.yaml#L393). I've introduced `videoObject` since it was used in one of the example. 

I could try to more consistent, or wait with this for later

Reply:
> so you would like to add imageURL as a separate slot?

yes

> For the future, I understand you would like to have something like this?

yes

re: audio and video

`audio`/`video`/`image` in schema.org are all similar in that they support multiple targets. however, `audio`/`video` don't have a `URL` option while `image` does. so for now i would file an issue if that is not handled in the current schema.